### ISSUE TITLE
Crash on "CTRL+SHIFT+V"

### ISSUE THREAD

**Issue by rkgdgames** [fpwong/BlueprintAssistWiki#62]
I just wanted to see the variables separeted, not on same node as I usually do. Then its crashing after new update. I hope that helps to fix.
```
Assertion failed: OwningNode [File:D:\RocketSync\4.26.0-14830424+++UE4+Release-4.26\Working\Engine\Source\Runtime\Engine\Classes\EdGraph/EdGraphPin.h] [Line: 482]

UE4Editor_Core!AssertFailedImplV() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Core\Private\Misc\AssertionMacros.cpp:104]
UE4Editor_Core!FDebug::CheckVerifyFailedImpl() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Core\Private\Misc\AssertionMacros.cpp:461]
UE4Editor_BlueprintAssist!FBAUtils::GetPinPos() [D:\build\++Portal\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\BlueprintAssist\Source\BlueprintAssist\Private\BlueprintAssistUtils.cpp:295]
UE4Editor_BlueprintAssist!FBAInputProcessor::DuplicateNodeForEachLink() [D:\build\++Portal\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\BlueprintAssist\Source\BlueprintAssist\Private\BlueprintAssistInputProcessor.cpp:1495]
UE4Editor_BlueprintAssist!TBaseRawMethodDelegateInstance<1,FBAInputProcessor const ,void __cdecl(void),FDefaultDelegateUserPolicy>::ExecuteIfSafe() [D:\RocketSync\4.26.0-14830424+++UE4+Release-4.26\Working\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:469]
UE4Editor_Slate!FUICommandList::ConditionalProcessCommandBindings() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Slate\Private\Framework\Commands\UICommandList.cpp:220]
UE4Editor_Slate!FUICommandList::ProcessCommandBindings() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Slate\Private\Framework\Commands\UICommandList.cpp:175]
UE4Editor_BlueprintAssist!FBAInputProcessor::HandleKeyDownEvent() [D:\build\++Portal\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\BlueprintAssist\Source\BlueprintAssist\Private\BlueprintAssistInputProcessor.cpp:543]
UE4Editor_Slate!FSlateApplication::InputPreProcessorsHelper::PreProcessInput() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Slate\Private\Framework\Application\SlateApplication.cpp:6900]
UE4Editor_Slate!FSlateApplication::ProcessKeyDownEvent() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Slate\Private\Framework\Application\SlateApplication.cpp:4246]
UE4Editor_Slate!FSlateApplication::OnKeyDown() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Slate\Private\Framework\Application\SlateApplication.cpp:4222]
UE4Editor_ApplicationCore!FWindowsApplication::ProcessDeferredMessage() [D:\Build\++UE4\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsApplication.cpp:2013]
UE4Editor_ApplicationCore!FWindowsApplication::DeferMessage() [D:\Build\++UE4\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsApplication.cpp:2642]
UE4Editor_ApplicationCore!FWindowsApplication::ProcessMessage() [D:\Build\++UE4\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsApplication.cpp:1046]
UE4Editor_ApplicationCore!FWindowsApplication::AppWndProc() [D:\Build\++UE4\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsApplication.cpp:878]
user32
user32
UE4Editor_ApplicationCore!FWindowsPlatformApplicationMisc::PumpMessages() [D:\Build\++UE4\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsPlatformApplicationMisc.cpp:130]
UE4Editor!FEngineLoop::Tick() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Launch\Private\LaunchEngineLoop.cpp:4748]
UE4Editor!GuardedMain() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Launch\Private\Launch.cpp:169]
UE4Editor!GuardedMainWrapper() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:137]
UE4Editor!WinMain() [D:\Build\++UE4\Sync\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:268]
UE4Editor!__scrt_common_main_seh() [d:\agent\_work\5\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288]
kernel32
ntdll
```

**Comment by fpwong** [comment]
I think I might know why this is crashing but I am still trying to replicate it. If you remember which nodes caused the crash that would be extremely helpful.

**Comment by rkgdgames** [comment]
Oh, it was a reference from a blueprint I've made. This was needed to get some info. 
It was a CoverObject and I was getting the location and setting some bools.

**Comment by fpwong** [comment]
This should be fixed in 3.2.6
